staging/src/k8s.io/api/go.mod

```diff
@@ -11,6 +11,7 @@ require (
 )
 
 replace (
+	golang.org/x/sys => golang.org/x/sys v0.0.0-20210603081109-ebe580a85c40
```

Comment:
not sure why we got all of these suddenly. I ran `hack/pin-dependency.sh` for the azure dep, then `hack/update-vendor.sh`, added changes to git. nothing else. go 1.16.5 darwin/amd64 from homebrew.

Reply:
must be https://github.com/kubernetes/kubernetes/pull/103010/files#diff-1b28dbbe1a53e9df49d0a1ea840546007c9b3d84d3bc36a3b2f43c8c8ce15a37R5

Reply:
answer: https://github.com/kubernetes/kubernetes/pull/103010#issuecomment-864300216